config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml

```diff
@@ -214,6 +214,13 @@ spec:
                             description: AuthClientPublicKeysURL specifies the public
                               keys URL
                             type: string
+                          ssoBaseURL:
+                            description: SSOBaseURL specifies the SSO base URL such
+                              as https://sso.redhat.com
+                            type: string
+                          ssoRealm:
+                            description: SSORealm specifies the SSO realm name
+                            type: string
```

Comment:
This is also already in master - no clue why it looks like that it's missing :man_shrugging: 
Make sure that you have the latest versions locally when running the `make generate` command